### PR TITLE
New blog post about ClusterRunner on Windows

### PR DESCRIPTION
**Pull request by aptxkid**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @aptxkid has signed the CLA. Thanks for the pull request!

**Comment by tjlee0909** [comment]
:+1:

File: _posts/2015-08-24-clusterrunner-on-windows.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+We make little assumptions on the target Windows machines being deployed. Windows doesn't support SSH natively. We provide an Ansible playbook so you can deploy ClusterRunner to a Windows Cluster with a single command!
+
+### CI on Appveyor
+
+![appveyor](/img/cr-appveyor.png)
+
+We are all very familir with Travis-CI, which provides Linux testing infrastructure for our Github projects. Appveyor is Travis-CI for Windows. We setted it up to prevent any regressions on Windows. All tests (unit, functional, integration) get kicked off on every Pull Request and Merge.
+
+### Eash Install
```
Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: Eash?

File: _posts/2015-08-24-clusterrunner-on-windows.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![deploy](/img/cr-windows-deploy.png)
+
+We make little assumptions on the target Windows machines being deployed. Windows doesn't support SSH natively. We provide an Ansible playbook so you can deploy ClusterRunner to a Windows Cluster with a single command!
+
+### CI on Appveyor
+
+![appveyor](/img/cr-appveyor.png)
+
+We are all very familir with Travis-CI, which provides Linux testing infrastructure for our Github projects. Appveyor is Travis-CI for Windows. We setted it up to prevent any regressions on Windows. All tests (unit, functional, integration) get kicked off on every Pull Request and Merge.
```
Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: "We setted" => "We set"

Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: "familir" => "familiar"

File: _posts/2015-08-24-clusterrunner-on-windows.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+---
+Tests are usually tightly coupled with the testing environment in some way. If you're building an iOS app, you test the app on iOS. ClusterRunner parallelize tests execution on a cluster of machines to bring you the fastest tests feedback possible. But it only supports Linux and Mac. What if you are building an app on Windows and want fastest test feecback on Windows as well?
+
+Today, I am glad to announce that ClusterRunner officially supports Windows as well as Linux and Mac! Here are some highlights.
+
+### Easy Ansible Deployable
+
+![deploy](/img/cr-windows-deploy.png)
+
+We make little assumptions on the target Windows machines being deployed. Windows doesn't support SSH natively. We provide an Ansible playbook so you can deploy ClusterRunner to a Windows Cluster with a single command!
```
Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: By "little assumptions" did you mean "few assumptions" or "tiny assumptions"?

File: _posts/2015-08-24-clusterrunner-on-windows.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+layout: news_item
+title: 'ClusterRunner Now Supports Windows!'
+date: 2015-08-24 10:00:00 -0800
+author: aptxkid
+categories: [release]
+---
+Tests are usually tightly coupled with the testing environment in some way. If you're building an iOS app, you test the app on iOS. ClusterRunner parallelize tests execution on a cluster of machines to bring you the fastest tests feedback possible. But it only supports Linux and Mac. What if you are building an app on Windows and want fastest test feecback on Windows as well?
```
Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: "parallelize" => "parallelizes"
"fastest tests feedback" => "fastest test feedback"

File: _posts/2015-08-24-clusterrunner-on-windows.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+### CI on Appveyor
+
+![appveyor](/img/cr-appveyor.png)
+
+We are all very familir with Travis-CI, which provides Linux testing infrastructure for our Github projects. Appveyor is Travis-CI for Windows. We setted it up to prevent any regressions on Windows. All tests (unit, functional, integration) get kicked off on every Pull Request and Merge.
+
+### Eash Install
+
+ClusterRunner tries to make installation really easy. We borrowed the idea from [Chocolatey](https://chocolatey.org/), that all you need to do to install ClusterRunner on Windows is to run this command on your Windows machine (not git clone required!):
```
Comment (josephharrington) [inline _posts/2015-08-24-clusterrunner-on-windows.markdown]: "not git clone" => "no git clone"

**Pull request merged**